### PR TITLE
Mrchem exercises

### PR DESCRIPTION
**Pull request by Andersmb**
- removed 3d viewer
- split exercises in 2 (not sure if the index renders correctly...)
- added solutions for exercise 1 and 2
- Implemented the grid_plotter into solution for exercise 1.

### CODE REVIEW

**Comment by stigrj** [comment]
I would prefer shorter, more catchy header names for the exercises, as they end up in the left menu panel

**Comment by Andersmb** [comment]
I am open to suggestions for the exercise names. I am feeling uncreative at the moment :)

**Comment by stigrj** [comment]
> I would remove the "Exercise XY:` from the title. I guess it's clear these are exercises, no?

Compromise: remove "Exercise", keep "XY:"

**Comment by robertodr** [comment]
Is there something missing in the non-solution notebooks? In the solution I see a **lot** of Python code. How am I supposed to know that that is needed to get through the exercise? There should be hints and scaffold throughout the non-solution notebook.

File: content/notebooks/how_to_run_mrchem.ipynb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (robertodr) [inline content/notebooks/how_to_run_mrchem.ipynb:1]: <p>Add the <code>$</code> for the prompt:</p><p> </p><pre class="ql-syntax" spellcheck="false">$ export OMP_NUM_THREADS=16
$ mrchem jobname.inp
</pre><p> </p><p>Personally, I prefer to do <code>$ OMP_NUM_THREADS=N mrchem jobname.inp</code> such that the environment is not polluted.</p><p>One question you might get: how do I put <code>mrchem</code> on my <code>PATH</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/multiwavelet-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: content/notebooks/mrchem-laptop-1.ipynb [hunk 2]
```diff
@@ -0,0 +1,299 @@
+{
```
Comment (robertodr) [inline content/notebooks/mrchem-laptop-1.ipynb:1]: <p>I think you can use LaTeX: *MW precision of $10^{-n}$*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/multiwavelet-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: content/notebooks/how_to_run_mrchem.ipynb (hunk 1 above)
Comment (Andersmb) [reply content/notebooks/how_to_run_mrchem.ipynb:1]: I thought they would need to do this due to the way it is set up with the conda environment?

File: content/notebooks/solutions/mrchem-laptop-1.ipynb [hunk 3]
```diff
@@ -0,0 +1,312 @@
+{
```
[the remark this answers is not in the document]
Comment (Andersmb) [reply content/notebooks/solutions/mrchem-laptop-1.ipynb:1]: <pre class="ql-syntax" spellcheck="false">`ModuleNotFoundError: No module named 'ipympl'
</pre><p> </p>

**Comment by stigrj** [comment]
ok, going in

**Pull request merged**